mm2src/coins/coin_errors.rs

```diff
@@ -24,7 +24,9 @@ pub enum ValidatePaymentError {
         "NumConversError",
         "UnexpectedDerivationMethod",
         "keys::Error",
-        "PrepareTxDataError"
+        "PrepareTxDataError",
+        "ethabi::Error",
+        "TryFromSliceError"
```

Comment:
`TryFromSliceError` is not needed and `PrepareTxDataError` should be used instead

Reply:
Didnt get you, if we remove `TryFromSliceError`, we will get errr for `_secret_hash.try_into()` cases
```
 the trait `std::convert::From<std::array::TryFromSliceError>` is not implemented for `coin_errors::ValidatePaymentError`
```

Also, instead of what?
> PrepareTxDataError should be used instead

Reply:
oh sorry, GitHub didn't clip the code I reviewed. I was actually talking about removing `TryFromSliceError`  in favor of `PrepareTxDataError`

Reply:
Oh I thought there was a `TryFromSliceError` variant. never mind no changes needed.